### ISSUE TITLE
Opera 12.1 applies @media screen\0

### ISSUE THREAD

**Issue by WebMechanic** [browserstrangeness/browserstrangeness.github.io#8]
The page only mentions MSIE8-11 and FF1.x but it's also red in Opera/12.1

Tested with The Final Presto
`Opera/9.80 (Windows NT 6.2; Win64; x64) Presto/2.12.388 Version/12.18`

**Comment by browserstrangeness** [comment]
Thank you, I usually don't test with Opera any more.

**Comment by WebMechanic** [comment]
Randomly, to check failing grid layouts :)
I'm always surprised how much "modern" CSS it actually supports.
'twas a fine browser; sad to see it gone. Vivaldi's no replacement.

**Comment by browserstrangeness** [comment]
Yep, for years it was nearly identically covered like Chrome for CSS in the releases.

**Comment by browserstrangeness** [comment]
I haven't really been following the development lately at all though.

**Comment by WebMechanic** [comment]
As for web standards there's virtually no difference between Chrome, Opera, Edgium, Vivaldi. 
Different user, devtool and privacy features but eventually all based on current Blink, V8 and Chromiun. 
Opera used to be innovative and Presto was a smooth and very fast rendering engine.
